docs/Other SDL Features/Remote Control Vehicle Features/index.md

```diff
@@ -188,11 +188,150 @@ if (sdlManager.getSystemCapabilityManager().isCapabilitySupported(SystemCapabili
+seatLocation.userLocation = <#Selected Seat#>;
+[self.sdlManager sendRequest:seatLocation withResponseHandler:^(__kindof SDLRPCRequest * _Nullable request, __kindof SDLRPCResponse * _Nullable response, NSError * _Nullable error) {
+    if(!response.success) { return; }
+    <#Seat Data Updated#>
+}];
+```
+
+##### Swift
+```swift
+let seatLocation = SDLSetGlobalProperties()
+seatLocation.userLocation = <#Selected Seat#>;
+sdlManager.send(request: setData, responseHandler: { (request, response, error) in
+    guard response?.success.boolValue == true else { return }
+    <#Seat Data Updated#>
+})
+```
+!@
+
+### Getting Remote Control Modules
+If the vehicle supports multiple modules of a module type you will have to pass in the `moduleID` along with the module type to control that module. To get a list of all modules you can check the `SDLGetSystemCapabilityResponse.remoteControlCapability` object. The `moduleID` is contained within `moduleInfo` in the remote control capability module.
+
+@![iOS]
+##### Objective-C
+```objc
+SDLGetSystemCapability *remoteControl = [[SDLGetSystemCapability alloc] initWithType: SDLSystemCapabilityTypeRemoteControl];
+[self.sdlManager sendRequest:remoteControl withResponseHandler:^(__kindof SDLRPCRequest * _Nullable request, __kindof SDLRPCResponse * _Nullable response, NSError * _Nullable error) {
+    if (!response.success) { return; }
+    SDLRemoteControlCapabilities *remoteCapabilities = ((SDLGetSystemCapabilityResponse *)response).systemCapability.remoteControlCapability;
+    <#Save Remote Capabilities#>
+}];
+```
+##### Swift
+```swift
+sdlManager.systemCapabilityManager.subscribe(toCapabilityType: .seatLocation, with: { capability in
+    guard let seats = capability.seatLocationCapability?.seats else { return }
+    <#Save Remote Capabilities#>
+})
+```
+!@
+
+With the saved remote capabilities struct you can build a UI/UX to display modules to the user. When the user selects the module you can send remote control RPCs using the `moduleInfo`s `moduleId` property. 
+
+!!! Note
+`ModuleID` is only required for Core v6.0+.  Examples will be shown later in the documentation on how to use `moduleID`.
+!!!
+
+@![iOS]
+##### Objective-C
+```objc
+NSString *selectedModuleID = <#SelectedModule#>.moduleInfo.moduleID;
+```
+
+##### Swift
+```swift
+let selectedModuleID = <#SelectedModule#>.moduleInfo?.moduleId
+```
+!@
+
+### Get Consent
+This feature is only available for Core v6.0+. Some OEMs may wish to ask the driver for consent before a user can control a module. This is typically done automatically by the HMI depending on the configuration. However, the `SDLGetInteriorVehicleDataConsent` RPC will alert the driver for consent in some OEMs if the module if not free "being used by another user"  and `allowMultipleAccess` "multiple users can access/set the data at the same time" is true. `allowMultipleAccess` is part of the `moduleInfo` in the module object.
```

Comment:
```suggestion
This feature is only available for Core v6.0+. Some OEMs may wish to ask the driver for consent before a user can control a module. The `SDLGetInteriorVehicleDataConsent` RPC will alert the driver for consent in some OEM head units if the module if not free (another user has control) and `allowMultipleAccess` (multiple users can access/set the data at the same time) is `true`. `allowMultipleAccess` is part of the `moduleInfo` in the module object.
```

Also are you sure this is true? Why would it be if it's not free **and** `allowMultipleAccess` is true? If `allowMultipleAccess` is `true`, in my mind, what does it matter if it's not free? Are you sure it's not "if it's not free and `allowMultipleAccess` is **`false`**?